andela/denethor-ah-backend
# #163559030 Server Environmental Setup

**What does this PR do?**
This PR sets up the environment for server-side and also configure the project to use ES

**Description of Task to be completed?**
Installation of babel for transpiling of javascript ES6 to ES5.
Create  .babelrc file for configuration options.
Installation of express framework
Setup index.js file to contain route and listening port.

**How should this be manually tested?**
Write code in ES6  in any file and verify server runs successfully. e.g write import instead of require.
Run the server on localhost:3000

**What are the relevant pivotal tracker stories?**
[163559030](https://www.pivotaltracker.com/story/show/163559030)
